Environment.Build.props

```diff
@@ -34,12 +34,14 @@
 
   <!-- This is used by the libraries -->
   <PropertyGroup Condition="'$(AndroidTargetFrameworks)' == ''">
-    <AndroidTargetFrameworks>MonoAndroid10.0;</AndroidTargetFrameworks>
+    <AndroidTargetFrameworks>MonoAndroid13.0;</AndroidTargetFrameworks>
   </PropertyGroup>
 
   <!-- Auto install any missing Android SDKs -->
   <PropertyGroup Condition="'$(CI)' == 'true'">
     <AndroidRestoreOnBuild Condition="'$(AndroidRestoreOnBuild)' == ''">False</AndroidRestoreOnBuild>
     <AcceptAndroidSDKLicenses Condition="'$(AcceptAndroidSDKLicenses)' == ''">False</AcceptAndroidSDKLicenses>
+    <AndroidSdkBuildToolsVersion>32.0.0</AndroidSdkBuildToolsVersion>
+    <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
```

Comment:
We shouldn't ever use this setting anymore, do you need it?

Was this something that worked around having Xamarin.Android 12.x installed? It would have moved to v12.0 if you tried to target v13.0.